Source/GUI/dvrescue/dvrescue/qwtquick2plot.cpp

```diff
@@ -763,7 +763,11 @@ void QwtQuick2PlotPicker::attach(QwtQuick2Plot *plot)
         this->setActive(pickerActive);
     });
 
+    #if QWT_VERSION >= 0x060200
     m_qwtPlotPicker->setAxes(QwtPlot::xBottom, QwtPlot::yLeft );
+    #else
+    m_qwtPlotPicker->setAxis(QwtPlot::xBottom, QwtPlot::yLeft );
+    #endif

```

Comment:
?, commented code and a repeated line. Does this have any effect?

Reply:
`#` is not comment but is preprocessor directive in C/C++ ;-).
And lines are not repeated (1 letter difference).